cob_cartesian_controller does not build on indigo armhf

Example: http://build.ros.org/view/Ibin_arm_uThf/job/Ibin_arm_uThf__cob_cartesian_controller__ubuntu_trusty_armhf__binary/222/console

```
04:05:19 [100%] Building CXX object CMakeFiles/cartesian_controller_node.dir/src/cartesian_controller_node.cpp.o
04:05:19 /usr/bin/arm-linux-gnueabihf-g++   -DROSCONSOLE_BACKEND_LOG4CXX -DROS_PACKAGE_NAME=\"cob_cartesian_controller\" -g -O2 -fstack-protector --param=ssp-buffer-size=4 -Wformat -Werror=format-security -DNDEBUG -D_FORTIFY_SOURCE=2  -I/tmp/binarydeb/ros-indigo-cob-cartesian-controller-0.6.14/obj-arm-linux-gnueabihf/devel/include -I/tmp/binarydeb/ros-indigo-cob-cartesian-controller-0.6.14/include -I/opt/ros/indigo/include    -o CMakeFiles/cartesian_controller_node.dir/src/cartesian_controller_node.cpp.o -c /tmp/binarydeb/ros-indigo-cob-cartesian-controller-0.6.14/src/cartesian_controller_node.cpp
04:06:25 Linking CXX executable devel/lib/cob_cartesian_controller/cartesian_controller_node
04:06:25 /usr/bin/cmake -E cmake_link_script CMakeFiles/cartesian_controller_node.dir/link.txt --verbose=1
04:06:25 /usr/bin/arm-linux-gnueabihf-g++   -g -O2 -fstack-protector --param=ssp-buffer-size=4 -Wformat -Werror=format-security -DNDEBUG -D_FORTIFY_SOURCE=2     CMakeFiles/cartesian_controller_node.dir/src/cartesian_controller_node.cpp.o  -o devel/lib/cob_cartesian_controller/cartesian_controller_node -rdynamic devel/lib/libcartesian_controller.so /opt/ros/indigo/lib/libtf.so /opt/ros/indigo/lib/libtf2_ros.so /opt/ros/indigo/lib/libactionlib.so /opt/ros/indigo/lib/libmessage_filters.so /opt/ros/indigo/lib/libroscpp.so -lboost_signals -lboost_filesystem /opt/ros/indigo/lib/libxmlrpcpp.so /opt/ros/indigo/lib/libtf2.so /opt/ros/indigo/lib/librosconsole.so /opt/ros/indigo/lib/librosconsole_log4cxx.so /opt/ros/indigo/lib/librosconsole_backend_interface.so -llog4cxx -lboost_regex /opt/ros/indigo/lib/libroscpp_serialization.so /opt/ros/indigo/lib/librostime.so -lboost_date_time /opt/ros/indigo/lib/libcpp_common.so -lboost_system -lboost_thread -lpthread -lconsole_bridge devel/lib/libtrajectory_interpolator.so devel/lib/libprofile_generator.so devel/lib/libcartesian_controller_utils.so /opt/ros/indigo/lib/libtf.so /opt/ros/indigo/lib/libtf2_ros.so /opt/ros/indigo/lib/libactionlib.so /opt/ros/indigo/lib/libmessage_filters.so /opt/ros/indigo/lib/libroscpp.so -lboost_signals -lboost_filesystem /opt/ros/indigo/lib/libxmlrpcpp.so /opt/ros/indigo/lib/libtf2.so /opt/ros/indigo/lib/librosconsole.so /opt/ros/indigo/lib/librosconsole_log4cxx.so /opt/ros/indigo/lib/librosconsole_backend_interface.so -llog4cxx -lboost_regex /opt/ros/indigo/lib/libroscpp_serialization.so /opt/ros/indigo/lib/librostime.so -lboost_date_time /opt/ros/indigo/lib/libcpp_common.so -lboost_system -lboost_thread -lpthread -lconsole_bridge -Wl,-rpath,/tmp/binarydeb/ros-indigo-cob-cartesian-controller-0.6.14/obj-arm-linux-gnueabihf/devel/lib:/opt/ros/indigo/lib: 
04:06:26 devel/lib/libprofile_generator.so: undefined reference to `vtable for TrajectoryProfileRamp'
04:06:26 devel/lib/libprofile_generator.so: undefined reference to `vtable for TrajectoryProfileSinoid'
04:06:26 collect2: error: ld returned 1 exit status
04:06:26 make[4]: *** [devel/lib/cob_cartesian_controller/cartesian_controller_node] Error 1
04:06:26 make[4]: Leaving directory `/tmp/binarydeb/ros-indigo-cob-cartesian-controller-0.6.14/obj-arm-linux-gnueabihf'
04:06:26 make[3]: *** [CMakeFiles/cartesian_controller_node.dir/all] Error 2
04:06:26 make[3]: Leaving directory `/tmp/binarydeb/ros-indigo-cob-cartesian-controller-0.6.14/obj-arm-linux-gnueabihf'
04:06:26 make[2]: *** [all] Error 2
04:06:26 make[2]: Leaving directory `/tmp/binarydeb/ros-indigo-cob-cartesian-controller-0.6.14/obj-arm-linux-gnueabihf'
04:06:26 dh_auto_build: make -j1 returned exit code 2
04:06:26 make[1]: *** [override_dh_auto_build] Error 2
04:06:26 make[1]: Leaving directory `/tmp/binarydeb/ros-indigo-cob-cartesian-controller-0.6.14'
04:06:26 make: *** [build] Error 2
04:06:26 dpkg-buildpackage: error: debian/rules build gave error exit status 2
04:06:26 E: Building failed
```

--- a/cob_cartesian_controller/src/trajectory_profile_generator/trajectory_profile_generator_ramp.cpp
+++ b/cob_cartesian_controller/src/trajectory_profile_generator/trajectory_profile_generator_ramp.cpp
@@ -31,7 +31,7 @@
 #include <cob_cartesian_controller/trajectory_profile_generator/trajectory_profile_generator_ramp.h>
 
 /* BEGIN TrajectoryProfileRamp ********************************************************************************************/
-inline bool TrajectoryProfileRamp::getProfileTimings(double Se, double te, bool calcMaxTe, cob_cartesian_controller::ProfileTimings& pt)
+bool TrajectoryProfileRamp::getProfileTimings(double Se, double te, bool calcMaxTe, cob_cartesian_controller::ProfileTimings& pt)
 {
     CartesianControllerUtils utils;
     double tv, tb = 0.0;
@@ -69,7 +69,7 @@ inline bool TrajectoryProfileRamp::getProfileTimings(double Se, double te, bool 
     return false;
 }
 
-inline std::vector<double> TrajectoryProfileRamp::getTrajectory(double se, cob_cartesian_controller::ProfileTimings pt)
+std::vector<double> TrajectoryProfileRamp::getTrajectory(double se, cob_cartesian_controller::ProfileTimings pt)
 {
     std::vector<double> array;
     unsigned int i = 1;

--- a/cob_cartesian_controller/src/trajectory_profile_generator/trajectory_profile_generator_sinoid.cpp
+++ b/cob_cartesian_controller/src/trajectory_profile_generator/trajectory_profile_generator_sinoid.cpp
@@ -31,7 +31,7 @@
 #include <cob_cartesian_controller/trajectory_profile_generator/trajectory_profile_generator_sinoid.h>
 
 /* BEGIN TrajectoryProfileSinoid ****************************************************************************************/
-inline bool TrajectoryProfileSinoid::getProfileTimings(double Se, double te, bool calcMaxTe, cob_cartesian_controller::ProfileTimings& pt)
+bool TrajectoryProfileSinoid::getProfileTimings(double Se, double te, bool calcMaxTe, cob_cartesian_controller::ProfileTimings& pt)
 {
     CartesianControllerUtils utils;
     double tv, tb = 0.0;
@@ -69,7 +69,7 @@ inline bool TrajectoryProfileSinoid::getProfileTimings(double Se, double te, boo
     return false;
 }
 
-inline std::vector<double> TrajectoryProfileSinoid::getTrajectory(double se, cob_cartesian_controller::ProfileTimings pt)
+std::vector<double> TrajectoryProfileSinoid::getTrajectory(double se, cob_cartesian_controller::ProfileTimings pt)
 {
     std::vector<double> array;
     unsigned int i = 1;

